core/classes/class-rest-api.php

```diff
@@ -316,6 +316,7 @@ function( $slug ) {
 		if ( $query->have_posts() ) {
 			foreach ( $query->posts as $post_id ) {
 				$event   = new Event( $post_id );
+				$venue_information = $event->get_venue_information();
```

Comment:
@jmarx Maybe set a default in case this function fails for whatever reason?

Reply:
As a follow-up, I realize that this is not needed as that `get_venue_information` method returns and empty venue by default.

Reply:
I would write a unit test for `get_venue_information` to ensure it doesn't fail in a way we don't expect.